include/sdf/Element.hh

```diff
@@ -112,6 +112,14 @@ namespace sdf
     /// \sa Element::SetRequired
     public: const std::string &GetRequired() const;
 
+    /// \brief Set if the element and children wer set or default in the file
+    /// \param[in] _value True if the element was set
+    public: void SetExplicitlySetInFile(const bool _value);
```

Comment:
Can you add an extra sentence stating that this is meant to be used by the parser to indicate whether the element was set by the user in the original file/source string?

Reply:
Done in 52cd85f7c78a366d72b266a842c372c48f1588ce